src/libraries/System.IO.Hashing/src/System/IO/Hashing/XxHash3.cs

```diff
@@ -572,13 +572,8 @@ private static ulong HashLengthOver240(byte* source, uint length, ulong seed)
                 const int BlockLen = StripeLengthBytes * StripesPerBlock;
                 int blocksNum = (int)((length - 1) / BlockLen);
 
-                uint offset = 0;
-                for (int i = 0; i < blocksNum; i++)
-                {
-                    Accumulate(accumulators, source + offset, secret, StripesPerBlock);
-                    ScrambleAccumulators(accumulators, secret + (SecretLengthBytes - StripeLengthBytes));
-                    offset += BlockLen;
-                }
+                Accumulate(accumulators, source, secret, StripesPerBlock, true, blocksNum);
+                var offset = BlockLen * blocksNum;
```

Comment:
var => actual type

... here and all the other places...

Reply:
Fixed by 2b32e8a